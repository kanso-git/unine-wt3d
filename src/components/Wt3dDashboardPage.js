import React from 'react';
import Wt3dList from './Wt3dList';
import Wt3dListFilters from './Wt3dListFilters';
import Wt3dsSummary from './Wt3dsSummary';

const Wt3dDashboardPage = () => (
  <div>
    <Wt3dsSummary />
    <Wt3dListFilters />
    <Wt3dList />
  </div>
);

export default Wt3dDashboardPage;
