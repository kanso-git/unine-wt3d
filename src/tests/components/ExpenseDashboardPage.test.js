import React from 'react';
import { shallow } from 'enzyme';
import Wt3dDashboardPage from '../../components/Wt3dDashboardPage';

test('should render Wt3dDashboardPage correctly', () => {
  const wrapper = shallow(<Wt3dDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
