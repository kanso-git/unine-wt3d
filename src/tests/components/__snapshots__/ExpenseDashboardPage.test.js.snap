// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Wt3dDashboardPage correctly 1`] = `
<div>
  <Connect(Wt3dsSummary) />
  <Connect(Wt3dListFilters) />
  <Connect(Wt3dList) />
</div>
`;
